src/FCharacter.cpp

```diff
@@ -140,6 +142,12 @@ bool FCharacter::facingLeft()
 void FCharacter::fire()
 {
 	m_weapons[m_currentWeapon].fire(/*m_facingLeft ? FDirection::left : FDirection::right*/);
+	m_load = false;
+}
+
+void FCharacter::loadWeapon()
+{
+	m_load = true;
```

Comment:
Still I had to add another method :/ , I just couldn't find a way to make m_load back to false automatically once it stops firing. It uses loadWeapon method in FGame.cpp